xplique/concepts/__init__.py

```diff
@@ -1,6 +1,14 @@
-"""
-Concept based methods
-"""
-
-from .cav import Cav
-from .tcav import Tcav
+"""
+Concept based methods
+"""
+
+from .cav import Cav
+from .tcav import Tcav
+from .ice import ICE
+from .ace import ACE
```

Comment:
It is really nice to have added all those methods! However, I did not see the related documentation. It has to be done before accepting this PR ;)

Reply:
Yes, I removed ACE and ICE, we will make a PR later with doc for those methods.